src/main/resources/schema.sql

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
What purpose of empty first line? It should be removed.

Reply:
fixed in